README.md

```diff
@@ -52,15 +52,15 @@ Syncfusion Essential XlsIO supports .xls, .xlsx, .xlsm, .xlsb, .xltm, .xltx, .xl
+Today, we provide 1700+ controls and frameworks for web ([Blazor](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [ASP.NET Core](https://www.syncfusion.com/aspnet-core-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [ASP.NET MVC](https://www.syncfusion.com/aspnet-mvc-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [ASP.NET WebForms](https://www.syncfusion.com/jquery/aspnet-webforms-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [Angular](https://www.syncfusion.com/angular-ui-components?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [React](https://www.syncfusion.com/react-ui-components?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [Vue](https://www.syncfusion.com/vue-ui-components?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), and [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples)), mobile ([Xamarin](https://www.syncfusion.com/xamarin-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), and [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples)), and desktop development ([Windows Forms](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [WPF](https://www.syncfusion.com/wpf-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [WinUI(Preview)](https://www.syncfusion.com/winui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples) and [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```

Comment:
remove preview for WinUI
<img width="982" alt="image" src="https://user-images.githubusercontent.com/61148953/207776679-1c216ef8-17d5-4082-b653-a52de16bf88a.png">

Should add MAUI platform to this list

Reply:
added maui platform and removed preview